_data/changelogs/component-icon.yml

```diff
@@ -2,6 +2,13 @@ title: Icon
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed deprecated xlink references.
+    summaryAdditional: All xlink:href references have been replaced with href and all xlmns:xlink references have been removed.
+    affectsMarkup: true
+    githubPr: 2951
```

Comment:
**question**: Should this point to #2926?

```suggestion
    githubPr: 2926
```

Reply:
@RachelCorsino @mejiaj 

**Question:** Are all user-facing guidance changes captured in #2926? Are there any changes in  uswds/uswds#6165 that would warrant a changelog as well? Such as component code previews being updated